tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Redis/RedisTags.cs

```diff
@@ -33,4 +34,32 @@ internal partial class RedisTags : InstrumentationTags
         [Metric(Trace.Metrics.RedisDatabaseIndex)]
         public double? DatabaseIndex { get; set; }
     }
+
+    internal partial class RedisV1Tags : RedisTags
+    {
+        private string _peerServiceOverride = null;
+
+        // Use a private setter for setting the "peer.service" tag so we avoid
+        // accidentally setting the value ourselves and instead calculate the
+        // value from predefined precursor attributes.
+        // However, this can still be set from ITags.SetTag so the user can
+        // customize the value if they wish.
+        [Tag(Trace.Tags.PeerService)]
+        public string PeerService
+        {
+            get => _peerServiceOverride ?? Host;
+            private set => _peerServiceOverride = value;
+        }
+
+        [Tag(Trace.Tags.PeerServiceSource)]
+        public string PeerServiceSource
+        {
+            get
+            {
+                return _peerServiceOverride is not null
+                           ? "peer.service"
+                           : "network.destination.name";
```

Comment:
I see that the integration already set `out.host` (but is not setting `network.destination.name`). when returning the source you have to check that the precursor is there.
It means that the logic in your case is something like
```
                return _peerServiceOverride is not null
                           ? "peer.service"
                           : Host is not null ? "out.host" : null;
```

Reply:
I was told that "network.destination.name" was identical to "out.host", so in all of the client tags where we use the "out.host" tag I've been setting the peer.service source to the value "network.destination.name". Can this stay the same or do we need to update all of the client integrations to change from "network.destination.name" to "out.host"?

Reply:
The meaning of those tag is identical indeed but according to the RFC the precursor that you are referring must exist. So actually if you use `out.host` you should refer to `out.host`. It's not vital because there is no integrity checks about that source but it should be changed if you can

Reply:
Ok. Maybe @pablomartinezbernardo could fix this PR and the ones that we've already done in that case (based on the metadata csv that should be easy)? WDYT?